tests/unit/src/kakarot/precompiles/test_precompiles.cairo

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: MIT
```

Comment:
in this file, you may want to test:
- `not_implemented_precompile` raises with the expected message
- `is_precompile` returns TRUE/FALSE appropriately
- `_exec_precompile` raises when `address` is greater than `LAST_PRECOMPILE_ADDRESS` (no particular warnings implemented, we should have a unknown memory location I guess)
- `run` returns an appropriate `ExecutionContext`

This may be out of the scope of this PR to add them all but at least `not_implemented_precompile` should be

Reply:
so the functions should be for example

```
func test_is_precompile_should_return_false_when_address_is_greater_than_last_precompile
func test_is_precompile_should_return_true_when_address_is_lower_than_last_precompile
func test_not_implemented_precompile_should_raise_with_detailed_error_message
func test_run_should_return_a_stopped_execution_context
func test_run_should_return_an_execution_context_with_input_ctx_as_calling_context
func test_run_should_return_an_execution_context_with_precompile_address_as_evm_address
```

I'd not test return data without mocking `_exec_precompile` and I don't know how to mock in our setting